components/engine_rocks/src/range_properties.rs

```diff
@@ -379,9 +187,7 @@ impl RangePropertiesExt for RocksEngine {
             let len = keys.len();
             keys = keys.into_iter().step_by(len / 10000).collect();
         }
-
         keys.sort();
-        keys.dedup();
```

Comment:
Why remove dedup?

Reply:
Cause it isn't correct. Consider the case that multiple SSTs have the same key point, if you deduplicate them, the size of the range will be underestimated. Though, it's better to duplicate the split keys, will do.